project_milestone/models/project_milestone.py

```diff
@@ -0,0 +1,45 @@
+from odoo import models, fields, api
+
+
+class ProjectMilestone(models.Model):
+    _name = 'project.milestone'
+    _order = 'sequence'
+
+    name = fields.Char(string="Title", required=True)
+    target_date = fields.Date(
+        string="Target Date",
+        required=False,
+        help="An target for when the Milestone is planned to be complete.")
+    progress = fields.Float(
+        string="Progress",
+        compute="_compute_milestone_progress",
+        store=True,
+        help="Percentage of Completed Tasks vs Incomplete Tasks.")
+    project_id = fields.Many2one('project.project', string="Project")
+    project_tasks = fields.One2many('project.task',
+                                    'milestone_id',
+                                    string="Project Tasks")
+    fold = fields.Boolean(string="KanBan Folded?")
+    sequence = fields.Integer(string="Sequence")
+
+    @api.model
+    def create(self, vals):
+        seq = self.env['ir.sequence'].next_by_code('project.milestone') or '/'
```

Comment:
Replace `or '/' ` with `or 0`-
Sequence is an Integer, assigning '/' will error.
If you add demo data this code will be implicitly tested.

Reply:
This has been updated in latest commit. Thanks.